WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -47,19 +50,32 @@ final class DashboardViewModel: ObservableObject {
     private let userDefaults: UserDefaults
     private let storeCreationProfilerUploadAnswersUseCase: StoreCreationProfilerUploadAnswersUseCaseProtocol
     private let themeInstaller: ThemeInstaller
+    private let storageManager: StorageManagerType
     private var subscriptions: Set<AnyCancellable> = []
 
     var siteURLToShare: URL? {
         if let site = stores.sessionManager.defaultSite,
-           site.isPublic, // only show share button if the site is public.
+           !site.isWordPressComStore || site.isPublic, // only show share button if it's a .org site or a public .com site
            let url = URL(string: site.url) {
             return url
         }
         return nil
     }
 
+    private lazy var ordersResultsController: ResultsController<StorageOrder> = {
+        let storageManager = ServiceLocator.storageManager
```

Comment:
This is redundant since you already injected `storageManager` in the initializer.

Reply:
Nice catch, handled in https://github.com/woocommerce/woocommerce-ios/pull/12521/commits/59a939c1c64aa3a5e4be1f8f0bb765c9eb1d932f